cdap-watchdog/src/main/java/co/cask/cdap/logging/serialize/LogSerializerUtil.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright © 2014-2017 Cask Data, Inc.
```

Comment:
It is actually a rename from `Util.java` to `LogSerializerUtil.java`. Anyway I can change it to 2017 only.